packages/terra-core-docs/src/terra-dev-site/test/tag/TagListCollapsedTags.test.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-alert */
+import React from 'react';
+import IconPerson from 'terra-icon/lib/icon/IconPerson';
+import IconBookmark from 'terra-icon/lib/icon/IconTile';
```

Comment:
With the recent changes in #3915 , Icons can now be imported as:

```suggestion
import { IconBookmark, IconPerson } from 'terra-icon';
```

Reply:
changes done. Thanks